src/schedule-and-details/index.jsx

```diff
@@ -309,6 +309,7 @@ const ScheduleAndDetails = ({ intl, courseId }) => {
             <StatefulButton
               onClick={handleUpdateValues}
               state={isQueryPending && 'pending'}
```

Comment:
The props `state` of the [StatefulButton component](https://paragon-openedx.netlify.app/components/statefulbutton/#props-api) must accept a `string`, please fix it

Reply:
let's do it next PR with fix console error